client/src/main/scala/org/apache/celeborn/client/CommitManager.scala

```diff
@@ -52,7 +47,7 @@ case class ShuffleCommittedInfo(
     committedSlaveStorageInfos: ConcurrentHashMap[String, StorageInfo],
     committedMapIdBitmap: ConcurrentHashMap[String, RoaringBitmap],
     currentShuffleFileCount: LongAdder,
-    commitPartitionRequests: util.Set[CommitPartitionRequest],
+    commitPartitionRequests: util.Set[PartitionLocation],
     handledCommitPartitionRequests: util.Set[PartitionLocation],
```

Comment:
committedPartitionLocation?

Reply:
> I will rename handledCommitPartitionRequests to handledCommitPartitionLocations as the partitionLocations are handled but may not committed yet.

yea, that's why I name it as `handled`.....I forgot this. `handledCommitPartitionLocations` is ok.